kong/plugins/basicauth2keyauth/handler.lua

```diff
@@ -0,0 +1,89 @@
+local resty_sha256 = require "resty.sha256"
+local to_hex = require "resty.string".to_hex
```

Comment:
Never seen this, is this loading the `to_hex` function from the `resty.string` module?  I would have expected to need to do `require("resty.string").to_hex` for that to work (haven't touched Lua in a long time though).